package.json

```diff
@@ -134,6 +134,11 @@
             "type": "string"
           },
           "markdownDescription": "Optional list of custom resolvers passed to Coursier when fetching metals dependencies.\n\nFor documentation on accepted values see the [Coursier documentation](https://get-coursier.io/docs/other-repositories).\n\nThe extension will pass these to Coursier using the COURSIER_REPOSITORIES environment variable after joining the custom repositories with a pipe character (|)."
+        },
+        "metals.superMethodLenses": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable/disable goto super method code lens"
```

Comment:
If this doesn't utilize markdown at all, I believe it can just be description as well.

Reply:
okay :+1: